moveos/moveos-verifier/src/build.rs

```diff
@@ -51,6 +269,34 @@ pub fn build_model(
     )
 }
 
+pub fn build_model_internal(
```

Comment:
Why `internal`? it is a public fn.

Reply:
Makes sense, the function name has been changed to `build_model_with_test_attr`.